setup.cfg

```diff
@@ -39,3 +41,21 @@ exclude =
 # module-attr: pytype cannot understand the namespace packages by Google.
 # attribute-error: Data classes in grpc module doesn't specify attributes.
 disable = "import-error,module-attr,attribute-error"
+
+[mypy]
+cache_dir = /dev/null
+disable_error_code = 
+    import,
+    name-defined
+files =
+    src/python/grpcio_admin/grpc_admin,
+    src/python/grpcio_channelz/grpc_channelz,
+    src/python/grpcio_csds/grpc_csds,
+    src/python/grpcio_health_checking/grpc_health,
+    src/python/grpcio_reflection/grpc_reflection,
+    src/python/grpcio_status/grpc_status,
+    examples/python/auth,
+    examples/python/helloworld
+exclude = (?x)(
```

Comment:
Why do these need to be excluded? There is a `.pyi` file generated alongside them that should provide type information.

Reply:
you are right. have removed exclude for these files